munininfluxdb/influxdbclient.py

```diff
@@ -221,19 +222,17 @@ def _upload_and_validate(measurement, tags, fields, packed_values):
             print("  {0} Connection to database \"{1}\" OK".format(Symbol.OK_GREEN, self.settings.influxdb['database']))
 
         if self.settings.influxdb['group_fields']:
-            """
-            In "group_fields" mode, all fields of a same plugin (ex: system, user, nice, idle... of CPU usage)
-             will be represented as columns of the same time series in InfluxDB.
-
-             Schema will be:
-                +----------------------+-------+----------+----------+-----------+
-                |   time_series_name   | col_0 |  col_1   |  col_2   | col_3 ... |
-                +----------------------+-------+----------+----------+-----------+
-                | domain.host.plugin   | time  | metric_1 | metric_2 | metric_3  |
-                | acadis.org.tesla.cpu | time  | system   | user     | nice      |
-                | ...                  |       |          |          |           |
-                +----------------------+-------+----------+----------+-----------+
-            """
+            # In "group_fields" mode, all fields of a same plugin (ex: system, user, nice, idle... of CPU usage)
```

Comment:
Replaced the string-literal with comments. Otherwise it's included in coverage which does not make sense.